sycl/source/backend/level_zero.cpp

```diff
@@ -25,10 +25,11 @@ __SYCL_EXPORT platform make_platform(pi_native_handle NativeHandle) {
   pi::PiPlatform PiPlatform;
   Plugin.call<PiApiKind::piextPlatformCreateWithNativeHandle>(NativeHandle,
                                                               &PiPlatform);
+  const auto &PlatformImpl =
+      getSyclObjImpl(detail::createSyclObjFromImpl<platform>(
+          std::make_shared<platform_impl>(PiPlatform, Plugin)));
```

Comment:
I think there is no need to do this work to get `PlatformImpl` because `getOrMakePlatformImpl()` is a static member function.  Instead, you should be able to call it directly below like:

```
return platform_impl::>getOrMakePlatformImpl(PiPlatform, Plugin);
```

Reply:
Done.